### PR TITLE
Excluded test locations from results

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by kkrampa** [comment]
@emord

File: custom/icds_reports/utils.py [hunk 1]
```diff
@@ -6298,18 +6540,23 @@ def get_medicine_kit_sector_data(config, loc_level):
...
 
     def get_data_for(filters):
         filters['month'] = datetime(*filters['month'])
-        return AggAwcMonthly.objects.filter(
+        queryset = AggAwcMonthly.objects.filter(
             **filters
         ).values(
             '%s_name' % loc_level
+        ).exclude(
+            state_id__in=get_test_state_locations_id(domain)
```
Comment (emord) [inline custom/icds_reports/utils.py]: This will always apply the exclusion even if show_test is true

File: custom/icds_reports/queries.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports import const
+
+
+def get_test_state_locations_id(domain):
```
Comment (emord) [inline custom/icds_reports/queries.py]: This is called a lot. it may be worthwhile to cache it for 5 minutes

File: custom/icds_reports/utils.py [hunk 3]
```diff
@@ -6550,18 +6803,23 @@ def get_infants_weight_scale_sector_data(config, loc_level):
...
```
Comment (emord) [inline custom/icds_reports/utils.py]: Will always apply the exclusion no matter the show_test

**Comment by kkrampa** [comment]
@calellowitz Could you merge this one?

**Pull request merged**